### DISCUSSION TITLE
How to support multi graph?

### DISCUSSION THREAD

**Discussion by leeyaunlong** [EINDEX/logseq-copilot#44] [Q&A]
How to support multi graphs?
I have 2 graphs, one for work , another for the life.
How to connect the 2 graphs with copilot?
- I can choice the graph when i save the content.
- In google search page, show 2 graphs result at side panel.

**Comment by EINDEX** [comment]
That depends on Logseq itself design. 

Unfortunately,  Logseq currently only supports one port for the API server. you cannot choose which graph to play the API server.

You can submit a feature request to Logseq side.

If they add a feature about this, we can apply it soon.
